browser/src/layer/vector/CanvasOverlay.ts

```diff
@@ -55,16 +96,16 @@ class CanvasOverlay {
 	private paths: Map<number, any>;
 	private bounds: cool.Bounds;
 	private tsManager: any;
-	private overlaySection: any;
-	private transform: OverlayTransform;
+	private overlaySection: CanvasSectionObject;
+	private transforms: TransformationsList;
 
 	constructor(mapObject: any, canvasContext: CanvasRenderingContext2D) {
 		this.map = mapObject;
 		this.ctx = canvasContext;
 		this.tsManager = this.map.getTileSectionMgr();
 		this.overlaySection = undefined;
 		this.paths = new Map<number, CPath>();
-		this.transform = new OverlayTransform();
+		this.transforms = new TransformationsList();
```

Comment:
The single `s` at the end is a very small difference. This wouldn't be a problem if there were no `transform` (without `s`) elsewhere. But since we are using both variables with a single character difference, sooner or later someone is going to have a typo and cause a bug.

Perhaps we should make this `trasnformList` to avoid confusion and accidental bugs ?

Reply:
This particular variable is a private member of the CanvasOverlay class. The other transform variable is a local variable of a different type (OverlayTransform). The typescript transpiler will complain if anyone uses `transforms` instead of `transform` or vice-versa. But I agree that renaming `transforms` to `transformList` makes it more distinctive and readable.